doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb

```diff
@@ -493,180 +539,373 @@
+    "# ax.plot(time,-acf, lw=2, color='b', ls='--')\n",
     "\n",
-    "1. Modify the code such that half of the particles are of <tt>type=1</tt>. Type 0 is implied for the remaining particles.\n",
-    "2. Specify Lennard-Jones interactions between type 0 particles with other type 0 particles, type 1 particles with other type 1 particles, and type 0 particles with type 1 particles (set parameters for <tt>system.non_bonded_inter[i,j].lennard_jones</tt> where <tt>{i,j}</tt> can be <tt>{0,0}</tt>, <tt>{1,1}</tt>, and <tt>{0,1}</tt>. Use the same Lennard-Jones parameters for interactions within a component, but use a different <tt>lj_cut_mixed</tt> parameter for the cutoff of the Lennard-Jones interaction between particles of type 0 and particles of type 1. Set this parameter to $2^{\\frac16}\\sigma$ to get de-mixing or to $2.5\\sigma$ to get mixing between the two components.\n",
-    "3. Record the radial distribution functions separately for particles of type 0 around particles of type 0, type 1 around particles of type 1, and type 0 around particles of type 1. This can be done by changing the <tt>ids1</tt>/<tt>ids2</tt> arguments of the <tt>espressomd.observables.RDF</tt> command. You can record all three radial distribution functions in a single simulation. It is also possible to write them as several columns into a single file.\n",
-    "4. Plot the radial distribution functions for all three combinations of particle types. The mixed case will differ significantly, depending on your choice of <tt>lj_cut_mixed</tt>. Explain these differences."
+    "from scipy.optimize import curve_fit\n",
+    "fitfn = lambda x,a,t: a*np.exp(-x/t)\n",
+    "popt,pcov = curve_fit(fitfn, time/(system.time_step*SAMPLING_INTERVAL), acf)\n",
+    "tcor = popt[1]\n",
+    "x = np.linspace(0,5)\n",
+    "ax.plot(x,np.exp(-x/tcor), lw=2, color='r', ls='-.')\n",
+    "# ax.set_yscale('log')\n",
+    "# ax.set_ylim(bottom=1e-3)\n",
+    "ax.set_xlim(0,5)\n",
+    "ax.tick_params(axis='both', labelsize=16)\n",
+    "ax.set_ylabel('normalized ACF', fontsize=20)\n",
+    "ax.set_xlabel(r'time / $\\tau_\\mathrm{S}$', fontsize=20)\n",
+    "ax.axhline(1./np.exp(1))\n",
+    "\n",
+    "print (\"Estimated correlation time:\", np.round(tcor,2))\n",
+    "```"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "solution2": "hidden"
+   },
    "source": [
-    "## Comparison with the literature\n",
-    "\n",
-    "Empirical radial distribution functions have been determined for pure fluids <a href='#[5]'>[5]</a>, mixtures <a href='#[6]'>[6]</a> and confined fluids <a href='#[7]'>[7]</a>. We will compare our distribution $g(r)$ to the theoretical distribution $g(r^*, \\rho^*, T^*)$ of a pure fluid <a href='#[5]'>[5]</a>."
+    "From the roughly exponential decay of the correlation function it dawns that only every second sample should be used to calculate statistically independent averages."
```

Comment:
My lecture on error estimation advises against this practice because it can lead to an overestimation of the standard error of the mean. I think it is better practice to carry out binning analysis, or better, autocorrelation function integration.